src/sentry/flags/providers.py

```diff
@@ -127,6 +148,30 @@ def handle_launchdarkly_event(
     ]
 
 
+def validate_launchdarkly_event(
+    request_data: bytes,
+    request_headers: HttpHeaders,
+    organization_id: int,
+) -> bool:
+    """Return "true" if the launchdarkly payload is valid."""
+    signature = request_headers.get("X-LD-Signature")
+    if signature is None:
+        return False
```

Comment:
Is auth through a secret always required for LD?

Reply:
Its not required by LD.  Its required by us.

Reply:
Line 108 of api.md says "Webhooks are optionally signed by their provider." Maybe we can mention there that we require use of a signature + secret?